Qt: Raise present window if launched with no new files

amhndu opened the pull request:
Previously, transmission-qt would only delegate and quit if new files
are added otherwise it would launch a new instance.
This would make it call PresentWindow on DBus if there are no files.

Closes #731

mikedld commented:
Qt client allows to start multiple instance because it's able to connect to remote sessions. If you want to prevent multiple instances creating multiple sessions on the same host (or better yet, with the same configuration directory), this is not the right way to do it.

amhndu commented:
Sorry, didn't see that issue previously.
So, if I understand correctly, it'd be ideal to show a dialog like the "Select Session" dialog, except one option says "Resume Local Session" ?

ckerr commented:
Discussion @ https://github.com/transmission/transmission/issues/255

amhndu commented:
As suggested in the issue:
* Added a new CLI option to open a new session.
* Added a file lock to ensure only one active session per config
* Implemented the interop function with COM, but couldn't test..

amhndu commented:
ping? @mikedld
